src/extension/alterschema/CMakeLists.txt

```diff
@@ -69,7 +69,8 @@ sourcemeta_library(NAMESPACE sourcemeta PROJECT core NAME alterschema
     linter/definitions_to_defs.h
 
     # Strict
-    strict/required_properties_in_properties.h)
+    strict/required_properties_in_properties.h
+    linter/non_applicable_enum_validation_keywords.h)
```

Comment:
This is not a strict rule. Can you list it above `# Strict` with the rest of `linter/` rules?